curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-destructuring-assignment-to-assign-variables-from-objects.english.md

```diff
@@ -63,9 +63,9 @@ const highTomorrow = HIGH_TEMPERATURES.tomorrow;
 
 // change code above this line
 
-console.log(yesterday) // should be not defined
-console.log(highToday); // should be 77
-console.log(highTomorrow); // should be 80
+console.log(yesterday) // Should not be defined
```

Comment:
Same issue as the previous file, I thought these were being removed.

Reply:
I added a new test to be able to remove these comments.